src/Verbosity.purs

```diff
@@ -0,0 +1,14 @@
+module Verbosity (Verbosity(..)) where
+
+import Prelude
+
+data Verbosity
```

Comment:
The default log levels are:
```purescript
data LogLevel = Trace | Debug | Info | Warn | Error
```
I think there are too many of the and that may confuse users, also there is no support for Silent more it seems. So I made our custom/simplified levels.

Reply:
Agree, it's both confusing and not that much useful in our case.

Just so you know, there will be a new Coercible class added to PureScript 0.14 (the realease seems imminent): https://github.com/purescript/purescript/pull/3351
Wonder how it could help here to coerce LogLevel <-> Verbosity 🤔 

(If you think it can help, it's worth adding a TODO here maybe?)